packages/replay/src/types/index.ts

```diff
@@ -2,3 +2,4 @@ export * from './performance';
 export * from './replay';
 export * from './replayFrame';
 export * from './rrweb';
+export type { EventType } from '@sentry-internal/rrweb';
```

Comment:
Hmm we have this duplicated [here](https://github.com/getsentry/sentry-javascript/blob/d35748d40bdbfc18953c71563b5045c6f63cf870/packages/replay/src/types/rrweb.ts#L6-L15) but I forgot why... might be because our fork of rrweb, it wasn't exported? cc @mydea